src/sentry/api/endpoints/organization_events.py

```diff
@@ -398,7 +398,20 @@ def _dashboards_data_fn(scopedDataset, offset, limit, scoped_query, dashboard_wi
                     else:
                         split_dataset = discover
 
-                    return _data_fn(split_dataset, offset, limit, scoped_query)
+                    response = _data_fn(split_dataset, offset, limit, scoped_query)
+
+                    if widget.discover_widget_split == DashboardWidgetTypes.ERROR_EVENTS:
+                        response["meta"][
+                            "discoverSplitDecision"
+                        ] = DashboardWidgetTypes.get_type_name(DashboardWidgetTypes.ERROR_EVENTS)
+                    elif widget.discover_widget_split == DashboardWidgetTypes.TRANSACTION_LIKE:
+                        response["meta"][
+                            "discoverSplitDecision"
+                        ] = DashboardWidgetTypes.get_type_name(
+                            DashboardWidgetTypes.TRANSACTION_LIKE
+                        )
+
+                    return response
```

Comment:
nit: Would we need to add this if the widget split is already saved on the widget model? In the UI we should prioritize the data from the widget model and if that isn't available, rely on the split decision. IMO the "discoverSplitDecision" meta should only be added if we make a decision by running queries in that request.

Reply:
Good point, I can remove this then for split widgets, thanks for calling that out!